test/web-platform-tests/to-upstream/uievents/getModifierState.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html dir="rtl">
```

Comment:
Is the "rtl"-direction relevant here, or merely copy-paste?

Reply:
It was a copypasta issue 😕 thanks for spotting!